modules/ROOT/nav.adoc

```diff
@@ -80,24 +75,26 @@
  ** xref:add-another-output-transform-studio-task.adoc[Add Another Output Target]
  ** xref:move-transformations-separate-file-studio-task.adoc[Move Transformations to Separate Files]
 
-* xref:eu-cloud-configuration.adoc[Configure EU Cloud in Studio]
-
-* xref:pce-configuration.adoc[Configure Private Cloud Edition in Studio]
- ** xref:pce4-configuration.adoc[Configure PCE 4 in Studio]
- ** xref:pce3-configuration.adoc[Configure PCE 3 in Studio]
- *** xref:pce-configuration-macos.adoc[MacOS]
- *** xref:pce-configuration-windows.adoc[Windows]
-
 * xref:login-platform-from-studio.adoc[]
  ** xref:set-credentials-in-studio-to.adoc[]
 
+* xref:set-anypoint-platform-cloud.adoc[Set Anypoint Platform Cloud]
+ ** xref:deploy-mule-application-task.adoc[Deploy a Mule Application to CloudHub]
+ ** xref:domain-support-concept.adoc[Domain Support]
+  *** xref:domain-studio-tasks.adoc[Design Mule Domains]
+ ** xref:eu-cloud-configuration.adoc[Configure EU Cloud in Studio]
+ ** xref:pce-configuration.adoc[Configure Private Cloud Edition in Studio]
+  *** xref:pce3-configuration.adoc[Configure PCE 3 in Studio]
+   **** xref:pce-configuration-macos.adoc[MacOS]
+   **** xref:pce-configuration-windows.adoc[Windows]
+  *** xref:pce4-configuration.adoc[Configure PCE 4 in Studio]
```

Comment:
I think put PCE 4 first, because it's new and shiny.

Reply:
OK! Thanks!